scripts/price_explorer/README.md

```diff
@@ -191,3 +191,35 @@ When adding a new resource to infracost, a `productFilter` has to be added that
     }
   }
   ```
+
+# MongoDB lookups
+
+Instead of using the scripts, you can also run `distinct` or `regex` queries on MongoDB to explore the prices.
+
+1. Install MongoDB version 4.
+2. Join our [Community Slack channel](https://www.infracost.io/community-chat), and send a message to Ali Khajeh-Hosseini, he'll send you a link so you can download a copy of his MongoDB dump.
+3. Import the prices to your local:
+
+  ```
+  tar -xvf cloudPricing.tar
+  mongorestore --gzip cloudPricing
```

Comment:
Needed to use "mongorestore -d cloudPricing --gzip cloudPricing", or other new db name after -d flag. Without that Mongo don't know what to do with files.